about.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About- OV Gaming Summit</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="overlay"></div>
+    <div id="mobile-menu" class="mobile-main-menu flex-col">
+        <img src="asset/fire.png" alt="">
+        <ul class="flex-col">
+            <li class="flex-row"><a href="index.html" class="menu-item">Home</a></li>
+            <li class="flex-row"><a href="about.html" id="link1" class="menu-item">About</a></li>
+            <li class="flex-row"><a href="#main-section" class="menu-item">Program</a></li>
+            <li class="flex-row"><a href="#" class="menu-item">Speakers</a></li>
+            <li class="flex-row"><a href="#sponsors" class="menu-item">Sponsors</a></li>
+            <li class="flex-row"><a href="#" class="menu-item">News</a></li>
+        </ul>
+    </div>
+    <header>
+        <div id="top-bar">
+            <ul>
+                <li>
+                    <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+                </li>
+                <li>
+                    <a href="#"><i class="fa-brands fa-twitter"></i></a>
+                </li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </div>
+        <div id="navbar">
```

Comment:
- [ ] Please always try to implement full changes given by your previous code reviewer, your previous code reviewer told you to also change the things in *about.html* which you changed in *index.html*. Kindly consider the usage of `nav` instead of `div` to make your code **semantically** more accurate which will indeed help you to make your website more accessible and more professional.